A test failed: ```[ERROR]   ControllerApiTest.testControllerApi:53 maintenance.json expected:<...eporter"},{"name":"C[]ontactInformationMai...> but was:<...eporter"},{"name":"C[loudTrialExpirer"},{"name":"C]ontactInformationMai...>```

`maintenance.json` must be updated.

--- a/controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/controller/responses/maintenance.json
+++ b/controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/controller/responses/maintenance.json
@@ -19,6 +19,9 @@
       "name": "CloudEventReporter"
     },
     {
+      "name": "CloudTrialExpirer"
+    },
+    {
       "name": "ContactInformationMaintainer"
     },
     {

